mruby-blake2b/src/mruby_blake2b.c

```diff
@@ -0,0 +1,86 @@
+#include <mruby.h>
+#include <mruby/class.h>
+#include <mruby/data.h>
+#include <mruby/string.h>
+
+#include "blake2b.h"
+
+static char const BLAKE2B_STATE_KEY[] = "$mrb_i_blake2b_state";
+
+static const struct mrb_data_type blake2b_state_type = {
+  BLAKE2B_STATE_KEY, mrb_free,
+};
+
+static mrb_value
+mrb_blake2b_init(mrb_state *mrb, mrb_value self)
+{
+  mrb_int outlen = 32;
+  // size_t outlen = 32;
+  blake2b_state *s;
+
+  mrb_get_args(mrb, "|i", &outlen);
+  // mrb_get_args(mrb, "s", &outlen);
```

Comment:
Can we just remove those commented lines?

Reply:
OK

Reply:
Done